### PR TITLE
link EBI data retrieval into the filter/plot/explore

### PR DESCRIPTION
**Pull request by wee-snufkin**
Adding another option for getting data into filter, plot, explore tutorial. Uses EBI data retrieval tool and some data manipulation to integrate well into the tutorial and its workflow. 
@nomadscientist

### CODE REVIEW

**Comment by nomadscientist** [comment]
Hi @wee-snufkin! Exciting!
This tutorial should go into the "Changing data formats & preparing objects" as its own (small) tutorial, rather than in the Filter/plot/explore one itself. Instead, just put a comment in the FPE that the data can be ingested from elsewhere and link to that.

**Comment by nomadscientist** [comment]
@Camila-goclowski it will also eventually be good to link this to the Seurat version you're doing

File: topics/single-cell/tutorials/EBI-retrieval/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+> 2. Rename {% icon galaxy-pencil %} output `Mito-counted AnnData for downstream analysis`
+>
+{: .hands_on}
+
+And that's all! What's even more exciting about AnnData Operations tool is that it automatically calculates a bunch of metrics, such as log1p_mean_counts, log1p_total_counts, mean_counts, n_cells, n_cells_by_counts, n_counts, pct_dropout_by_counts, total_counts. Amazing, isn't it?
+
+# Conclusion
+Now you can use this object as input for the [Filter, Plot, Explore tutorial]({% link topics/single-cell/tutorials/scrna-case_basic-pipeline/tutorial.md %}) and its associated workflow! 
+
+Even though this tutorial was designed specifically to modify the AnnData object to be compatible with the subsequent tutorial, it also shows useful tools that you can use for your own, independent data analysis. You can find the [workflow]() and the [answer key history](). However, if you want to use the workflow from this tutorial, you have to keep in mind that different datasets may have different column names. So you have to check them first, and only then you can modify them.
```
Comment (bgruening) [inline topics/single-cell/tutorials/EBI-retrieval/tutorial.md:181]: Here is something missing `[workflow]() and the [answer key history]()`

File: topics/single-cell/tutorials/scrna-case_basic-pipeline/tutorial.md [hunk 2]
```diff
@@ -108,6 +108,12 @@ You can access the data for this tutorial in multiple ways:
 >
 {: .hands_on}
 
+4. **Importing data from EBI Single Cell Expression Atlas**
+
+You can also pull the data from publicly available [Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc/home). You can simply access the dataset we are working on by using the tool *EBI SCXA Data Retrieval* with experiment id of [E-MTAB-6945](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6945/downloads). This [short tutorial]() will show you how to use this tool and modify the output so that it's compatible with this tutorial and its workflow.
```
Comment (bgruening) [inline topics/single-cell/tutorials/scrna-case_basic-pipeline/tutorial.md:113]: here as well `[short tutorial]()`

**Comment by wee-snufkin** [comment]
Thanks @bgruening, I'll fill the links once EBI retrieval tool starts working again and allows me to generate the final history and workflow!

**Comment by bgruening** [comment]
@wee-snufkin can I help to fix the tool?

**Comment by wee-snufkin** [comment]
> @wee-snufkin can I help to fix the tool?

It's all sorted! :)

File: topics/single-cell/tutorials/EBI-retrieval/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+Now we can create an AnnData object!
+
+# Creating AnnData object
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 
+> 1. {% tool [Scanpy Read10x](toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_read_10x/scanpy_read_10x/1.8.1+galaxy9) %}
+> 2. Make sure you are using version **1.8.1+galaxy9** of the tool (change by clicking on {% icon tool-versions %} Versions button):
```
Comment (bgruening) [inline topics/single-cell/tutorials/EBI-retrieval/tutorial.md:116]: @wee-snufkin what is broken in the latest version?

Should we hide this latest version?

Comment (wee-snufkin) [reply topics/single-cell/tutorials/EBI-retrieval/tutorial.md:116]: That's the error message when I'm running this step with 1.8.1+galaxy93
image

If it's not a quick fix, we might want to hide the latest version

Comment (bgruening) [reply topics/single-cell/tutorials/EBI-retrieval/tutorial.md:116]: This error is fixable. We just need an older numpy version. e.g. numpy==1.21

Can you try this on EU please with the latest version? If this works I will fix the upstream package.

Comment (nomadscientist) [reply topics/single-cell/tutorials/EBI-retrieval/tutorial.md:116]: @pcm32 This is a matter of tool developer skills, if I'm not mistaken? Are you able to advise here?

Comment (pcm32) [reply topics/single-cell/tutorials/EBI-retrieval/tutorial.md:119]: We don't seem to get this error on instances using singularity (although not sure at the exact wrapper version). Those newer versions only have changes in the galaxy wrapper, the dependency had remained the same for a long time I think

**Review by pavanvidem — APPROVED** [review]
Looks good to me.

File: topics/single-cell/tutorials/EBI-retrieval/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+      - scrna-case_alevin-combine-datasets
+
+follow_up_training:
+  -
+    type: "internal"
+    topic_name: single-cell
+    tutorials:
+        - scrna-case_basic-pipeline
+
+tags:
```
Comment (nomadscientist) [inline topics/single-cell/tutorials/EBI-retrieval/tutorial.md:40]:
```suggestion
tags:
 - data management
 - data import
```

File: topics/single-cell/tutorials/EBI-retrieval/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+  - transcriptomics
```
Comment (nomadscientist) [inline topics/single-cell/tutorials/EBI-retrieval/tutorial.md:41]:
```suggestion
  - data import
```

It's not strictly bulk transcriptomics so hence we've removed these from anything where that's the case

File: topics/single-cell/tutorials/EBI-retrieval/tutorial.md (hunk 3 above)
Comment (nomadscientist) [reply topics/single-cell/tutorials/EBI-retrieval/tutorial.md:162]: So what's going on with this? I'm reviewing this following @wee-snufkin instructions in the tutorial to use the older version. If this issue gets fixed, we can update it I guess

**Review by nomadscientist — APPROVED** [review]
The workflow + workflow testing will need minor updates based on how there's now the Seurat option, and the tool versioning on the Scanpy tools is still an issue, but as it's written the tutorial runs well now!

File: topics/single-cell/tutorials/EBI-retrieval/tutorial.md (hunk 3 above)
Comment (nomadscientist) [reply topics/single-cell/tutorials/EBI-retrieval/tutorial.md]: I've copied this discussion over to here so we don't lose it in the merged PR: https://github.com/galaxyproject/tools-iuc/issues/5636

**Pull request merged**